examples/flows/failed-consent/definition.yaml

```diff
@@ -0,0 +1,20 @@
+StartAt: HelloState
+States:
+  HelloState:
+    Type: Action
+    Comment: >-
+      This state will always result in an INACTIVE run, requiring user
+      intervention to complete.
+    ActionUrl: https://actions.automate.globus.org/hello_world
+    Parameters:
+      echo_string: urn:globus:auth:scope:groups.api.globus.org:all
```

Comment:
Not `echo_string` here but `required_dependent_scope`

Reply:
Oh good catch! Using `echo_string` makes the name for this Flow irreleavnt.